python/cudf/cudf/tests/test_numerical.py

```diff
@@ -384,3 +385,24 @@ def test_to_numeric_error(data, errors):
         got = cudf.to_numeric(data, errors=errors)
 
         assert_eq(expect, got)
+
+
+@pytest.mark.parametrize("dtype", NUMERIC_TYPES)
+@pytest.mark.parametrize("input_obj", [[1, cudf.NA, 3]])
+def test_series_construction_with_nulls(dtype, input_obj):
+    dtype = np.dtype(dtype)
+    # numpy case
+
+    expect = pd.Series(input_obj, dtype=cudf_dtypes_to_pandas_dtypes[dtype])
+    got = cudf.Series(input_obj, dtype=dtype).to_pandas(nullable=True)
+
+    assert_eq(expect, got)
+
+    # Test numpy array of objects case
+    np_data = np.array(
+        [dtype.type(v) if v is not cudf.NA else cudf.NA for v in input_obj]
+    )
+
+    expect = pd.Series(input_obj, dtype=cudf_dtypes_to_pandas_dtypes[dtype])
```

Comment:
Did you mean to use `np_data` here instead of `input_obj`?  `np_data` not being used is causing flake8 to fail.

Reply:
Yes - nice catch. Fixed